openapi/components/schemas/Subscription.yaml

```
@@ -109,6 +109,14 @@ properties:
       If this value is `true`, recurring settings are ignored.
     type: boolean
     default: false
+  isTrialConverted:
+    description: |-
+      Specifies if an order had a trial conversion.
+      Order considered to has a trial conversion if at least one successful payment made on the order after a trial.
```

suggestion: to help make more concise
```suggestion
      Specifies if an order had trial conversion.
      Trial conversion is when at least one successful payment is made on the order after a trial.
```